src/fu-engine.c

```diff
@@ -5761,6 +5761,13 @@ fu_engine_plugin_rules_changed_cb (FuPlugin *plugin, gpointer user_data)
 	}
 }
 
+static void
+fu_engine_plugin_config_changed_cb(FuPlugin *plugin, gpointer user_data)
+{
+	FuEngine *self = FU_ENGINE(user_data);
+	fu_engine_set_status(self, FWUPD_STATUS_SHUTDOWN);
```

Comment:
I think you need a non-verbose message here for the reason to shut down the daemon for the journal.